src/main/java/com/redislabs/redisai/RedisAI.java

```diff
@@ -4,9 +4,10 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Paths;
-import java.util.ArrayList;
+import java.util.*;
```

Comment:
@filipecosta90 why?

Reply:
changed towards importing ArrayList and List specifically